src/main/java/me/coley/recaf/ui/MainMenu.java

```diff
@@ -235,10 +237,16 @@ private void addLibrary() {
 		fcLoadApp.setInitialDirectory(config().getRecentLoadDir());
 		List<File> files = fcLoadApp.showOpenMultipleDialog(null);
```

Comment:
Why don't you add check before opening file dialog?

Reply:
@xxDark, oh, i didn't think about it at all, i'll change location of check with new commit